modules/local/get_chrom_sizes.nf

```diff
@@ -0,0 +1,24 @@
+process GET_CHROM_SIZES {
+    tag "$fai"
+
+    conda (params.enable_conda ? "conda-forge::coreutils=8.31" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gnu-wget:1.18--0' :
```

Comment:
am using this instead of docker.io/library/alpine because when executing script block, nextflow appends bash shebang line and alpine image doesn't run on bash shell.

Reply:
OK, wouldn't it be cleaner with a minimal ubuntu container? Not that it matters :)

Reply:
If it's smaller, then yup! I can't find the singularity ubuntu container so I'll just stick with what it is now 😃

Reply:
The ubuntu container is definitely larger